README.md

```diff
@@ -12,6 +12,13 @@ Do you want to help testing pre-releases of OSM Surveyor?
 
 ![Screenshot of the map that displays annotations](screenshot.png?raw=true)
 
+# Getting started with development
+
+1. Install [Bundler][5]: `gem install bundler`
+2. Install the Gems (such as Cocoapods and fastlane): `bundle install`
+3. Install the dependencies using Cocoapods: `bundle exec pod install`
```

Comment:
4. `bundle exec pod install` will installed the dependencies from the existing Podfile

Reply:
Thanks for the feedback!  I've added a paragraph above the list that points out the `Podfile`.